flaxandteal/coral-arches
# Align user in pkg and root

### What does this PR do?

Makes sure that the `pkg` extension for user, and the installed copy in root match, so that Arches does not try and copy one over the other when deployed.

It also ensures we use the stripped-down user widget, with a select2 dropdown, rather than the not-working language-based one (bearing in mind that Django does not have i18n fields for usernames/emails).

### How can I test this PR?

Ensure that a DB reset does not fail with a permissions issue for `user.js`

For the widget, as an admin, go to a person editing page on an instance with the Person model loaded and confirm that the user card shows a dropdown of the users on the system.

### Who can review this PR?

@OwenGalvia or @taylorn01
